From 7fdc802085826a969fb49f4775e5067808e2a228 Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Mon, 17 Dec 2018 17:55:04 +0800
Subject: [PATCH 409/706] drm: hdp: move DP Firmware loading step to U-boot
 from kernel

In order to be consistent with I.MX, so move the loading step
to U-boot.

Signed-off-by: Wen He <wen.he_1@nxp.com>
(cherry picked from commit 66eec6684cf87d63b62abca9c10c738b24427b17)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-dp.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-dp.h b/drivers/gpu/drm/imx/hdp/imx-dp.h
index a920ede99b27..ba21d3a4a869 100644
--- a/drivers/gpu/drm/imx/hdp/imx-dp.h
+++ b/drivers/gpu/drm/imx/hdp/imx-dp.h
@@ -7,9 +7,6 @@
 #ifndef _IMX_DP_H_
 #define _IMX_DP_H_
 
-/* Loading FW in the kernel */
-#define DEBUG_FW_LOAD
-
 void dp_fw_load(state_struct *state);
 int dp_fw_init(state_struct *state);
 void dp_mode_set(state_struct *state,
-- 
2.17.1

